core/src/debug_ui.rs

```diff
@@ -21,27 +23,33 @@ use std::sync::{Arc, Weak};
 use swf::{Color, Rectangle, Twips};
 use weak_table::PtrWeakKeyHashMap;
 
+use self::handle::DomainHandle;
```

Comment:
Wouldn't this be nicer up there on line 13 together with the other handles?

Reply:
I guess, but I think things like import order should just be left up to `rustfmt`

Reply:
Absolutely, but it won't switch between `crate::` and `self::`, because that's semantically not the same. In this case, it won't matter (AFAIU), but switching would make it look nicer.